flood_forecast/evaluator.py

```diff
@@ -200,14 +220,21 @@ def infer_on_torch_model(
             columns=list(range(num_prediction_samples)),
             dtype="float",
         )
-        df_prediction_samples.iloc[history_length:] = prediction_samples
+        if decoder_params is not None:
+            if "probabilistic" in decoder_params:
+                df_prediction_samples.iloc[history_length:] = prediction_samples[0]
```

Comment:
Is this because it is returning a tuple?

Reply:
Yes, but I think I'll remove the probabilistic bit here altogether because it doesn't make sense to have the option for a bayesian confidence interval for a probabilistic dnn, because that sort of defeats the whole point of adding a layer that gives a confidence interval organically. Let me know what you think.